tools/generate-helpers/pg-table-bindings/store.go.tpl

```diff
@@ -410,6 +410,8 @@ func (s *storeImpl) UpsertMany(ctx context.Context, objs []*{{.Type}}) error {
 func (s *storeImpl) Count(ctx context.Context) (int, error) {
 	defer metrics.SetPostgresOperationDurationTime(time.Now(), ops.Count, "{{.TrimmedType}}")
 
+    var sacQueryFilter *v1.Query
```

Comment:
you do not need to declare here.

Reply:
if I don't declare it here, I'd need another branch to set it to nil